package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "lint": "npx eslint .",
+
+    "install:connex": "cd packages/connex && npm install",
```

Comment:
This project is a monorepo managed by lerna, these install command are not necessarily required